### ISSUE TITLE
System-wide config file

### ISSUE THREAD

**Issue by xordspar0** [tmate-io/tmate#172]
Instead of hard-coding the config file to ~/.tmate.conf, could we add multiple places to check for the config file? It would be useful to me to be able to have a company-local package that installs a config file to something like /etc/tmate.conf with our own settings.

**Comment by nviennot** [comment]
Instead of hardcoding to /etc/tmate.conf, maybe an env variable like`TMATE_CONF_PATH=/etc/tmate.conf` would work better?
This env variable can be set system wide in /etc/profile.

**Comment by xordspar0** [comment]
Right, an environment variable would be ideal of multiple scenarios, including when deploying in a Docker container.

**Comment by nviennot** [comment]
For docker, `-f /path/to/config` is good enough

**Comment by xordspar0** [comment]
Oh, sorry. I forgot what we were talking about. This is for the client. I don't run the client in a docker container. But I guess running in a docker container and mounting in a config file is another option.

**Comment by lnxbil** [comment]
> Instead of hardcoding to /etc/tmate.conf, maybe an env variable like`TMATE_CONF_PATH=/etc/tmate.conf` would work better?

It's different and breaks how `tmux` would do it. Using `/etc/tmate.conf` was the first thing I tried and maybe others would try it that way, too. It's simple Unix/Linux convention over configuration. The environment variable is fine and can be used too, yet I would do it additionally.
